### ISSUE TITLE
ModuleNotFoundError: No module named 'pipsi' after install

### ISSUE THREAD

**Issue by carc1n0gen** [mitsuhiko/pipsi#118]
I just installed pipsi for the first time, and afterward when I run `pipsi` I get

```
10:00 $ pipsi
Traceback (most recent call last):
  File "/Users/evansca/.local/bin/pipsi", line 7, in <module>
    from pipsi import cli
ModuleNotFoundError: No module named 'pipsi'
```

I found a closed issue that looked similar that was solved by reinstalling python, but that didn't work for me.  Let me know if you need more information from me.

**Comment by bjoernpollex** [comment]
Same here, just reinstalled `pipsi`, was working fine before, but now if fails with that message.

**Comment by bjoernpollex** [comment]
Hm, this seems to be a `virtualenv` problem. Looking at `~/.pipsi/bin/pipsi`, I get this:

```python
#!/usr/local/bin/python3

# -*- coding: utf-8 -*-
import re
import sys

from pipsi import cli

if __name__ == '__main__':
    sys.argv[0] = re.sub(r'(-script\.pyw?|\.exe)?$', '', sys.argv[0])
    sys.exit(cli())
```

The problem is that the `#!` doesn't use the Python interpreter from the `pipsi` virtual environment, but the one used for installing pipsi. That environment does indeed not have the package available. I could fix it by changing the line like this:

```
#!/Users/bjoern/.pipsi/virtualenv/pipsi/bin/python3
```

I'm running on 3.6.3 (installed from homebrew) with virtualenv 15.1.0.

**Comment by bjoernpollex** [comment]
The issue seems to be triggered by `virtualenv` being preferred when its present. #123 offers a fix.

**Comment by bjoernpollex** [comment]
This seems to be fixed in #142, so I'd suggest closing this issue.
